jujugui/static/gui/src/app/components/profile/model-list/model-list.js

```diff
@@ -0,0 +1,256 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+/**
+  Model list React component used to display a list of the users models in
+  their profile.
+*/
+class ProfileModelList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      loadingModels: false,
+      models: null
+    };
+  }
+
+  componentWillMount() {
+    this._fetchModels(this.props.facadesExist);
```

Comment:
Should facadesExist be called controllerConnected or just apiConnected? I guess this component is interested in whether the API is connected, not caring too much about the facade concept.